reference/constraints/NotBlank.rst

```diff
@@ -94,4 +94,13 @@ message
 
 This is the message that will be shown if the value is blank.
 
+trim
+~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+If ``true`` and if the given value is a ``string``, then the :phpfunction:`trim`
+PHP function will be used in order to ignore leading and trailing whitespace
+characters while checking if it is valid.
```

Comment:
Why does the NotBlank constraint only have `trim` whereas all the others have `ltrim` and `rtrim`?

Reply:
These options were replaced by the `normalizer` option. The patch PR has changed a lot since it's creation so I'm waiting a bit more to update the docs.